hubspot-style/pom.xml

```diff
@@ -62,7 +62,6 @@
     <dependency>
       <groupId>com.hubspot.rosetta</groupId>
       <artifactId>RosettaAnnotations</artifactId>
-      <scope>test</scope>
```

Comment:
This is a bit of an unfortunate dependency to pick up, do you think this can go away at some point in the future or are we stuck with it forever?

Reply:
Nope, that did not work. looks like its unavoidable to include `@RosettaAnnotation` in hubspot-immutables or something like `@ImmutableInherited` (from hubspot-immutables) into Rosetta. We could seperate out the RosettaAnnotation annotation into a seperate RosettaAnnotationsBase module, to cut down on the dependency size? :shrug:

Reply:
I'm less worried about the size of the dep tree, moreso just feels to take a hard dependency on Rosetta here. But if it can't be avoided I guess we can just go with it